### DISCUSSION TITLE
How to rewrite nextjs app paths after extraction.

### DISCUSSION THREAD

**Discussion by ym-project** [vercel/next.js#29540] [Help]
Hi!

I would talk about static files extraction.
I have a nextjs app. I execute `next build && next extract` and serve html files as static. But there is a problem: I have `/index.html`, `/page1.html` and etc. but extracted application doesn't change his own paths and redirect to `/`, `/page1`, ... without extension.

And after page reloading my web server shows me 404 error because there is no a page. I don't want to add rewrite rule to web server to handle `/page` -> `/page.html`

How to fix it?

**Comment by wwwdepot** [comment]
You need to configure your HTTP server to strip the html extension.

**Reply by ym-project** [reply]
That is I can't resolve this problem via nextjs CLI or config file's option?
